keras/applications/mobilenet.py

```diff
@@ -0,0 +1,275 @@
+'''MobileNet models for Keras.
+# Reference
+- [MobileNets: Efficient Convolutional Neural Networks for
+   Mobile Vision Applications](https://arxiv.org/pdf/1704.04861.pdf))
+'''
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import warnings
+
+from keras.models import Model
+from keras.layers.core import Activation, Dropout, Reshape
+from keras.activations import relu
+from keras.layers.convolutional import Convolution2D, DepthwiseConvolution2D
+from keras.layers.pooling import GlobalAveragePooling2D, GlobalMaxPooling2D
```

Comment:
Don't import submodules like `pooling` they are no longer meant to be used. Also avoid importing specific layers. Instead import `layers`